.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"prettier.configPath": ".prettierrc.js"
```

Comment:
I'm really not sure why this is needed 😕 , did your vscode didn't find the config by itself?

(
as far as I can tell prettier should work fine without any sort of path configuration? https://prettier.io/docs/en/configuration.html )

Reply:
If one has a global VS Code setting for the prettier config, this will override that. Not that I recall ever actually setting it, but I happen to have a global setting to use `.prettierrc` for the config path, so for me adding a project-scoped settings file will override the global setting I have.

Reply:
that definitely doesn't sound right 😕 (https://github.com/prettier/prettier-vscode#configuration)

I'm pretty sure that the local config should get picked up before the global one and that this shouldn't be necessary

anyways I am not sure if there are downsides in setting this so I guess we can just add it if it solves issues you're having locally

Reply:
I thought the same thing, but my VS Code global settings are meaning it is only trying to find `.prettierrc` instead of `.prettierrc.js`, so the project-scoped VS Code settings file overrides that global setting I have.

There aren't any downsides to specifying this in a project-scoped settings file - it just means that the setting takes precedence over any global settings.

![image](https://user-images.githubusercontent.com/10815538/228820644-f92fa84a-22c8-4311-ab89-156b886270b3.png)